frontend/src/app/pages/products/products.component.ts

```diff
@@ -169,27 +174,27 @@ export class ProductsComponent implements OnInit, OnDestroy {
 
     this.router.navigateByUrl(this.createNewUrl(newQuery));
   }
-
+//done
```

Comment:
what does it mean? Maybe there should be TODO, FIXME here. Writing TODO, FIXME you make comment visible in tree of todos in TodoTree vs code extension